mono_repo/lib/src/commands/travis.dart

```diff
@@ -205,7 +205,16 @@ EXIT_CODE=0
 for PKG in \${PKGS}; do
   echo -e "\\033[1mPKG: \${PKG}\\033[22m"
   pushd "\${PKG}" || exit \$?
-  pub upgrade --no-precompile || exit \$?
+
+  PUB_EXIT_CODE=0
+  pub upgrade --no-precompile || PUB_EXIT_CODE=\$?
+
+  if [[ \${PUB_EXIT_CODE} -ne 0 ]]; then
+    EXIT_CODE=1
```

Comment:
Should this be `PUB_EXIT_CODE` instead of 1?

Reply:
I tried that, but got a weird error – like it was converted to a string or something. 🤷‍♂